### DISCUSSION TITLE
SQL Server Express append log files?

### DISCUSSION THREAD

**Discussion by slthom** [olahallengren/sql-server-maintenance-solution#614] [General]
When I run the solution, I only get the most recent log file in my output folder.  I log to table as well.  Is it possible to have the log files appended with a date and time like the backup files?  Here's the cmd I am running.

sqlcmd -E -S SQLEXPRESS -d maintenanceplan -Q "EXECUTE dbo.DatabaseBackup @Databases = 'USER_DATABASES', @Directory = 'D:\Z\Database\backups', @BackupType = 'FULL', @Verify = 'Y', @CleanupTime = 168, @LogToTable = 'Y'" -b -o D:\Z\Database\backups\Maintenanceplan\Log\DatabaseBackup.txt

**Comment by olahallengren** [comment]
I think that you would have to write a small cmd or PowerShell script for that.
